### DISCUSSION TITLE
Developer documentation?

### DISCUSSION THREAD

**Discussion by romain145** [bennymeg/Fabrication-Toolkit#103] [Q&A]
Hi, I would like to help fix an issue I raised for this plugin.
Where can I find the KiCAD Plugin developper documentation? Is there a way to develop with a debugger? I'm using VSCode.

**Comment by bennymeg** [comment]
KiCad Documantation:
- [v6](https://docs.kicad.org/doxygen-python-6.0/namespacepcbnew.html)
- [v7](https://docs.kicad.org/doxygen-python-7.0/namespacepcbnew.html)

You cannot attach vs code debugger to KiCad.
There is an integrated python console into KiCad that you can use to test fragments of code.
